include/seqan3/alignment/configuration/align_config_aligned_ends.hpp

```diff
@@ -69,7 +69,7 @@ struct seq_end_gap_base
 };
 
 // ----------------------------------------------------------------------------
-// first_seq_leading
+// front_end_first
```

Comment:
does `first` correspond to the "first" sequence or sequence 1? Because you call free ends `first_ends_free` and now `first` is at the end.. maybe make this consistent? But I have no overview on all the member names

Reply:
i see your concern.